daemon/state.go

```diff
@@ -130,16 +130,26 @@ func (d *Daemon) regenerateRestoredEndpoints(state *endpointRestoreState) {
 	// they have finished to rebuild after being restored.
 	epRegenerated := make(chan bool, len(state.restored))
 
+	// Insert into endpoint manager so it can be regenerated when calls to
+	// TriggerPolicyUpdates() are made. This must be done synchronously (i.e.,
+	// not in a goroutine) because regenerateRestoredEndpoints must guarantee
+	// upon returning that endpoints are exposed to other subsystems via
+	// endpointmanager.
+	endpointmanager.InsertEndpoints(state.restored)
```

Comment:
Quick question: Are the endpoints in the correct state (maybe `StateRestoring`) here when being inserted to `endpointmanager`. I see that you simply moved the insertion step out of the asynchronous go routine, but wanted to confirm that we do not need to do anything additional in terms of setting state to something like `StateRestoring` or equivalent.

Reply:
thank!